README.md

```diff
@@ -1,28 +1,24 @@
-<h1 align="center">Alacritty</h1>
 <p align="center">
     <img width="200" alt="Alacritty Logo" src="extra/logo/compat/alacritty-term+scanlines.png">
 </p>
 
-Alacritty is the fastest terminal emulator in existence. Using the GPU for
-rendering enables optimizations that simply aren't possible without it.
-Alacritty currently supports macOS, Linux, BSD, and Windows.
+<h1 align="center">Alacritty - A fast, cross-platform, OpenGL terminal emulator</h1>
 
 <p align="center">
   <img width="600"
        alt="Alacritty running vim inside tmux"
```

Comment:
Really minor nit, but is this still "vim inside tmux"? I guess you could be hiding the tab bar thing, but it doesn't seem necessary to call out the tmux part when it's not even visible.

Reply:
It's not.

Reply:
https://github.com/alacritty/alacritty/pull/4623 should fix it.